addons/im_livechat/static/src/js/website_livechat_window.js

```diff
@@ -16,12 +16,29 @@ var LivechatWindow = AbstractThreadWindow.extend({
      * @override
      * @param {im_livechat.im_livechat.LivechatButton} parent
      * @param {im_livechat.model.WebsiteLivechat} thread
+     * @param {Object} [options={}]
+     * @param {string} [options.headerBackgroundColor]
+     * @param {string} [options.titleColor]
      */
-    init: function (parent, thread) {
+    init: function (parent, thread, options={}) {
```

Comment:
since you change this line, could use short-hand method name `init(parent, thread, options={}) {`